Sources/XMTP/Codecs/ReplyCodec.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReplyCodec.swift
+//
+//
+//  Created by Naomi Plasterer on 7/26/23.
+//
+
+import Foundation
+
+
+public let ContentTypeReply = ContentTypeID(authorityID: "xmtp.org", typeID: "reply", versionMajor: 1, versionMinor: 0)
+
+public struct Reply: Codable {
+    public var reference: String
+    public var content: Any
+    public var contentType: ContentTypeID
+}
```

Comment:
I think I'm going to need someone better at Swift to help me make this Codable. I tried `JSONSerialization` without success. cc @nakajima